tests/approaches/test_gnn_policy_approach.py

```diff
@@ -200,3 +200,5 @@ def _solved_classifier(s, o):
                                         test_task.goal_holds,
                                         max_num_steps=CFG.horizon)
     assert "Option plan exhausted" in str(e)
+    # TODO: test that shooting does not infinitely hang in the case where the
```

Comment:
`git grep "MockOptionModel"` might help, and then just the existence of that test would be sufficient (otherwise unit tests would take infinite time)

Reply:
thanks, yeah I saw that, but can you think of an easy way to get the GNN approach to use the mock option model?

Reply:
...it should be clear from the above no? you can see lines like `approach._option_model = _MockOptionModel1(_simulator)  # pylint: disable=protected-access`

Reply:
thx sry was being dum